app/presenters/nsm/check_answers/read_only_report.rb

```diff
@@ -124,6 +124,12 @@ def further_information_section
 
       private
 
+      def case_disposal_form
+        return CaseDisposalCard.new(claim) if claim.include_youth_court_fee.nil?
+
+        CaseCategoryCard.new(claim)
+      end
+
```

Comment:
I think the only thing I'm worried about here is if we end up unable to finish this feature before 6th december, which to be honest is fast approaching now. We may end up being left with submissions that were submitted on or after december 6th that could have technically been elligible but we won't know because they were never able to enter the correct case outcome. So it might make sense to display something to indicate as such - but I think that's a bigger design question so just adding it as a comment rather than an actual review

Reply:
Something to raise with James and Mary. All the provider tickets are in review except CYA and the CW changes are being worked on now, and while Mary is QAing provider we'll all be focused on CW so it seems very likely we'll hit 6th december.

Still raise it tomorrow standup though.